src/tribler/core/components/reporter/exception_handler.py

```diff
@@ -111,6 +112,7 @@ def unhandled_error_observer(self, _, context):
                 should_stop=should_stop
             )
             self.logger.error(f"Unhandled exception occurred! {reported_error}\n{reported_error.long_text}")
+            process_locker.set_error(exc=exception)
```

Comment:
When an exception is caught, it is added to the description of the current process